llvm/test/CodeGen/AMDGPU/prevent-fmul-hoist-ir.ll

```diff
@@ -0,0 +1,447 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -passes=simplifycfg -verify-machineinstrs -hoist-common-insts=true -mtriple=amdgcn-- --fp-contract=fast -mcpu=gfx1030 < %s | FileCheck -check-prefix=GFX -check-prefix=FP-CONTRACT-FAST %s
```

Comment:
-verify-machineinstrs does nothing with opt. Is hoist-common-insts available as a pass parameter instead of a cl::opt? 

Also can you pre-commit the test

Reply:
Updated hoist-common-insts as pass parameter and created PR for pre-committing test #121925